app/cmd/add_replica.go

```diff
@@ -41,6 +41,12 @@ func AddReplicaCmd() cli.Command {
 				Value:    5,
 				Usage:    "HTTP client timeout for replica file sync server",
 			},
+			cli.IntFlag{
+				Name:     "grpc-timeout-seconds",
```

Comment:
Should we make the name more specific? For example, `rebuilding-grpc-timeout-seconds`?

Reply:
This is inside the AddReplicaCmd so I think it is only for rebuilding. WDYT?

Btw, we also use it inside snapshotClone command so I feel that rebuilding-grpc-timeout-seconds doesn't seem cover that case. WDYT?

Reply:
Got it. Then, using `grpc-timeout-seconds` is fine.